installer/templates/phx_assets/daisyui.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Bundled by jsDelivr using Rollup v2.79.2 and Terser v5.37.0.
```

Comment:
We should have a comment at the top that shows how to update it, something like: `curl -sLo daisyui.js https://esm.run/daisyui@5/index.js`